mirage/apt/read_apt_xml.py

```diff
@@ -927,16 +926,40 @@ def read_globalalignment_template(self, template, template_name, obs, prop_param
             else:
                 long_pupil = 'CLEAR'
 
-        # Repeat for the number of exposures + 1
-        for j in range(n_exp + 1):
+        for fgs_conf in ga_fgs_configs:
+            fgs_grps = fgs_conf.find(ns + 'Groups').text
+            fgs_ints = fgs_conf.find(ns + 'Integrations').text
+
+        # TODO read guider detector requirement from GuideStarID special requirement in this case
+        guider_det_num = 1  # PLACEHOLDER ONLY
```

Comment:
We do this in MAGIC so if you are interested you can find APT query here: https://github.com/spacetelescope/jwst_magic/blob/3b9f9933cb4bcdb49f8874cd45e1a2972b023a87/jwst_magic/masterGUI.py#L1263
Specifically lines 1333-1355:
`# Pull: List of Observations for the CAR > Specific Obs > Special Requirements Info (sr)
        namespace_tag = '{http://www.stsci.edu/JWST/APT}'
        observation_list = tree.find(namespace_tag + 'DataRequests').findall('.//' + namespace_tag + 'Observation')
        observation = observation_list[obs_number - 1]  # indexes from 1
        sr = [x for x in observation.iterchildren() if x.tag.split(namespace_tag)[1] == "SpecialRequirements"][0]

        # Try to pull the Guide Star information
        try:
            gs = [x for x in sr.iterchildren() if x.tag.split(namespace_tag)[1] == "GuideStarID"][0]
        except IndexError:
            self.lineEdit_normalize.setText('')
            LOGGER.error("Master GUI: This observation doesn't have a Guide Star Special Requirement")
            raise ValueError("This observation doesn't have a Guide Star Special Requirement")

        # Pull out the guide star ID and the guider number
        gs_id = [x for x in gs.iterchildren() if x.tag.split(namespace_tag)[1] == "GuideStar"][0].text
        guider = [x for x in gs.iterchildren() if x.tag.split(namespace_tag)[1] == "Guider"][0].text

        # Account for if the guider is written as "guider 1" or "guider1"
        if not isinstance(guider, int):
            guider = guider.lower().replace(' ', '').split('guider')[1]

        LOGGER.info('Master GUI: APT has been queried and found guide star {} and guider {}'.format(gs_id, guider))
`